templates/workspace_services/mlflow/terraform/variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "workspace_id" {}
+variable "tre_id" {}
+
+variable "resource_id" {}
+
+variable "mgmt_acr_name" {}
+variable "mgmt_resource_group_name" {}
+
+variable "image_name" {
+  type        = string
+  description = "value"
+  default     = "mlflow-server"
+}
+variable "image_tag" {
```

Comment:
Can image tag be calculated from the verion.txt file as per https://github.com/microsoft/AzureTRE/blob/5919af2553d6c9b3257c5897526c03805421a645/templates/workspace_services/gitea/terraform/main.tf#L86 ?

Reply:
Applied. Will test and push shortly.